paddle/fluid/operators/nccl/nccl_gpu_common.cc

```diff
@@ -19,7 +19,11 @@ namespace paddle {
 namespace platform {
 namespace {
 // TODO(panyx0718): Where to destroy them.
+#ifdef PADDLE_WITH_HIP
+std::unique_ptr<std::vector<rcclComm_t>> global_comms;
```

Comment:
Does RCCL have the same API as NCCL2?

I cannot found `GroupStart()/GroupEnd()` in this [header](https://github.com/ROCmSoftwarePlatform/rccl/blob/master/inc/rccl.h). 

BTW, instead of writing annoying macros in many source files. Could we write an abstract layer for NCCL and RCCL? 

The abstract layer could be even not part of Paddle and could be an individual open source project.  That project would be widely useful.

Reply:
AFAIK, RCCL doens't have GroupStart()/GroupEnd() .
How about make paddle/fluid/platform/dynload/nccl.h the abstract layer, and unify APIs such as dynload::rcclCommInitAll and dynload::ncclCommInitAll into dynload::cclCommInitAll ?